src/RestControllers/AuthRestController.php

```diff
@@ -225,6 +238,7 @@ public function isValidToken($tokenRaw)
         if (($_SESSION['api'] == "port") || ($_SESSION['api'] == "pofh")) {
             // For patient portal api
             $_SESSION['pid'] = $tokenResult['pid'];
+            $_SESSION['puuid'] = $tokenResult['uuid'];
```

Comment:
btw, there will be a time in the distant future, where we will be using $puuid instead of $pid throughout OpenEMR :)
(ie. a baby is born :) )

Reply:
This scares me a bit!:) I have to agree with Rod on this ie becomes difficult communicating uuids intra office.
IMO, uuids are most useful for interop and MRNs should be comprehensible in normal language.

Reply:
Don't worry, plan is only for internal/interop. The mrn (public facing per institution, which is pubpid) will be unaffected. Trust me...

![tenor (21)](https://user-images.githubusercontent.com/278968/82945085-31821f00-9f51-11ea-8f47-41645490e2c6.gif)